spec/fixtures/unpatched_versions/issues.json

```diff
@@ -1,275 +1,338 @@
-    },
-    {
-        "categories": [
-            "Security"
-        ],
-        "check_name": "Insecure Dependency",
-        "content": {
-            "body": "**Advisory**: CVE-2015-7499\n\n**Criticality**: Medium\n\n**URL**: https://groups.google.com/forum/#!topic/ruby-security-ann/Dy7YiKb_pMM\n\n**Solution**: upgrade to >= 1.6.7.2"
-        },
-        "description": "Nokogiri gem contains a heap-based buffer overflow vulnerability in libxml2\n",
-        "fingerprint": "65fefc6b16a09d76d5c76b03247a48e1",
-        "location": {
-            "lines": {
-                "begin": 74,
-                "end": 74
-            },
-            "path": "Gemfile.lock"
-        },
-        "remediation_points": 50000,
-        "severity": "normal",
-        "type": "Issue"
-    },
-    {
-        "categories": [
-            "Security"
-        ],
-        "check_name": "Insecure Dependency",
-        "content": {
-            "body": "**Advisory**: CVE-2015-7578\n\n**URL**: https://groups.google.com/forum/#!topic/rubyonrails-security/uh--W4TDwmI\n\n**Solution**: upgrade to ~> 1.0.3"
-        },
-        "description": "Possible XSS vulnerability in rails-html-sanitizer",
-        "fingerprint": "b344012bca53e8faaafbf0f943776ea0",
-        "location": {
-            "lines": {
-                "begin": 97,
-                "end": 97
-            },
-            "path": "Gemfile.lock"
-        },
-        "remediation_points": 500000,
-        "severity": "normal",
-        "type": "Issue"
-    },
-    {
-        "categories": [
-            "Security"
-        ],
-        "check_name": "Insecure Dependency",
-        "content": {
-            "body": "**Advisory**: CVE-2015-7580\n\n**URL**: https://groups.google.com/forum/#!topic/rubyonrails-security/uh--W4TDwmI\n\n**Solution**: upgrade to ~> 1.0.3"
-        },
-        "description": "Possible XSS vulnerability in rails-html-sanitizer",
-        "fingerprint": "10818805098b10f2fe6e181f0961445c",
-        "location": {
-            "lines": {
-                "begin": 97,
-                "end": 97
-            },
-            "path": "Gemfile.lock"
-        },
-        "remediation_points": 500000,
-        "severity": "normal",
-        "type": "Issue"
-    },
-    {
-        "categories": [
-            "Security"
-        ],
-        "check_name": "Insecure Dependency",
-        "content": {
-            "body": "**Advisory**: OSVDB-126747\n\n**URL**: https://github.com/mishoo/UglifyJS2/issues/751\n\n**Solution**: upgrade to >= 2.7.2"
-        },
-        "description": "uglifier incorrectly handles non-boolean comparisons during minification",
-        "fingerprint": "118e36b455317d002e7fde24873ecc40",
-        "location": {
-            "lines": {
-                "begin": 118,
-                "end": 118
-            },
-            "path": "Gemfile.lock"
-        },
-        "remediation_points": 5000000,
-        "severity": "normal",
-        "type": "Issue"
-    }
+  {
```

Comment:
the history for this file will be a bit obfuscated with the spacing change. is it necessary?  does this match our style?